curriculum/structure/superblocks/full-stack-developer.json

```
@@ -378,6 +378,7 @@
             "lab-gradebook-app",
             "lecture-the-var-keyword-and-hoisting",
             "lab-falsy-remover",
+            "lab-title-case-converter",
```

```suggestion
            "lab-title-case-converter",
            "lab-falsy-remover",
```
the order on the spreadsheet is not the same you have here